spotbugsTestCases/src/java7/sfBugs/Bug3458246.java

```diff
@@ -1,37 +0,0 @@
-package sfBugs;
```

Comment:
The bug is https://sourceforge.net/p/findbugs/bugs/1003/, shouldn't we move this to Java 8 directory instead of deleting it?

Reply:
OK I will.